flang/include/flang/Parser/parse-tree.h

```diff
@@ -3738,6 +3738,33 @@ struct OmpOrderClause {
   std::tuple<std::optional<OmpOrderModifier>, Type> t;
 };
 
+// REF: [5.1:217-220], [5.2:293-294]
```

Comment:
let us keep all changes related to interop at one place in this file. currently directive definition is in different place and the init clause def is in different place.